docs/dg/dev/internationalization-and-multi-store/add-new-store-in-multi-db-setup.md

```diff
@@ -0,0 +1,163 @@
+
+### Cloud environment and monitoring
+* Think of and adjust monitoring tools and APM (such as NewRelic, CloudWatch) to accommodate the new store. Check that all alerts and metrics are correctly configured to monitor the health and performance of the new store alongside existing ones.
+* Think of adjusting AWS Services such as S3, introducing buckets for the new store(s).
+
+### Front-end Considerations
+* Consider any other activities related to the above epics that might impact the deployment. For instance, front-end separation might be a significant task, requiring layout adjustments between different stores and possible adjustments on the API side.
+
+## Step-by-Step Procedure to release a new store(s)
+
+Follow [this guideline](/docs/scos/dev/technical-enhancement-integration-guides/integrate-multi-database-logic.html#define-databases) as a generic technical guideline, which is a requirement from the cloud team, to successfully deploy your new store in a multi-db setup.
+
+### Local Setup
+#### New store configuration
+
+* Define a new database and the store in the deploy file, following [that guide](/docs/ca/dev/multi-store-setups/add-and-remove-databases-of-stores.html#remove-the-configuration-of-the-database). As a result you should have:
+  * new database in regions.<region_name>.services.databases 
+  * new store in regions.<region_name>.stores 
+  * new domains in groups.<region_name>.applications  
+* Adjust stores.php with the configurations, relevant for your new store, following generic technical guideline. 
+* Prepare data import configurations and data files, specific to the new store. 
+* Adjust the local environment setup as needed, including configurations and environment variables. Examples: frontend router configuration, code bucket configuration, creating new backoffice users. 
+* Document all the steps you have done, to make sure they are repeatable in the future.
+
+#### Running Initial Setup
+
+* Boot the new configurations using the appropriate setup tool. 
+* Execute commands to initialize the database and services for the new store. 
+* Verify that the new store’s database is correctly initialized with demo data.
+
+#### Setting up additional deployment recipes
+
+It is convenient to create additional deployment install recipes (located under config/install folder) to setup a new and delete an existing stores, for testing purposes. Below is an example of such setup that proves to be working well on prcatice. We took the existing folder structure, and EU folder as a base, but you can introduce your structure:
+
+1. config/install/EU/setup-store.yml - contains everything needed to do a minimal setup of a new store(s):
+```
+env:
+  NEW_RELIC_ENABLED: 0
+command-timeout: 7200
+stores:
+  - { STORES_GO_HERE }
```

Comment:
if I'm not mistaken, they have to be placed each on a separate line

Reply:
yes, you're right - looks confusing. Fixed!